sceptre/hooks/asg_scaling_processes.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from sceptre.hooks import Hook
+from sceptre.exceptions import InvalidHookArgumentTypeError
+from sceptre.exceptions import InvalidHookArgumentSyntaxError
+from sceptre.exceptions import InvalidHookArgumentValueError
+
+
+class ASGScalingProcesses(Hook):
+    """
+    A command to resume or suspend autoscaling group scaling processes. This is
+    useful as scheduled actions must be suspended when updating stacks with
+    autoscaling groups.
+    This command supports all autoscaling scaling processes.
+    Example configuration:
+    hooks:
+      before_update:
+        - !asg_scaling_processes suspend::ScheduledActions
+    Full documentation on the suspend and resume processes
+    http://docs.aws.amazon.com/autoscaling/latest/userguide/as-suspend-resume-processes.html
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(ASGScalingProcesses, self).__init__(*args, **kwargs)
+
+    def run(self):
+        """
+        Either suspends or resumes any scaling processes on all autoscaling
+        groups with in the current stack.
+        """
```

Comment:
Also add a rst docstring raises bit at the end... `:raises: InvalidHookArgumentSyntaxError`